src/bindings/io/uv/example/exampleio_uv.c

```diff
@@ -0,0 +1,131 @@
+/**
+ * @file
+ *
+ * @brief Example program for io_uv binding
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ * This program uses two operations:
+ * - The "input" operation is a file descriptor watcher that waits for
+ *   STDIN_FILENO (stdin) to become readable.
+ *   Since input is buffered, this typically happends when the user enters some
+ *   text and presses return.
+ * - The "output" operation is a timer that prints the last read data every
+ *   second.
+ */
+#include <assert.h>
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <kdb.h>
+#include <kdbhelper.h>
+#include <kdbio.h>
+#include <kdbio_uv.h>
+
+#include <uv.h>
+
+#define BUFFER_LENGTH 255
+#define ONE_SECOND 1000
+#define MIN(a, b) (((a) < (b)) ? (a) : (b))
```

Comment:
Why implement as macro?

Reply:
No reason at all - some result on stackoverflow suggested this macro... :)
I switched to a function.